ai-agent/README.md

```
@@ -0,0 +1,105 @@
+Apply the pgvector schema once (from repo root):
+
+```bash
+psql "$DATABASE_URL" -f database/vector_schema.sql
+```
+
+> **Supabase on an IPv4 network:** use the **connection pooler** host, not the
+> direct `db.<ref>.supabase.co` host. The direct host is IPv6-only, so on
+> IPv4-only machines psycopg fails with `connection is bad: no error details
+> available`. Use the pooler instead (session mode, port 5432):
+>
+> ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)